mw-webapp/src/component/table/columns.tsx

```diff
@@ -15,60 +15,99 @@ const DEFAULT_SUMMARY_TIME = 0;
 
 const columnHelper = createColumnHelper<DayReport>();
 
-const getObjectArrayItem = (
-  arrayItem: JobDone | PlanForNextPeriod | CurrentProblem,
-  getFullItem?: string,
-) => {
-  return JSON.stringify(arrayItem) === "{}" ? (
-    <div />
-  ) : (
-    <div key={arrayItem.uuid}>
-      {getFullItem}
-    </div>
+/**
+ * Render cell item based on array item
+ */
+const renderObjectArrayItem = (arrayItem: JobDone | PlanForNextPeriod | CurrentProblem, getFullItem?: string) => {
+  return (
+    (JSON.stringify(arrayItem) === "{}") ?
+      <div />
+      :
+      <div key={arrayItem.uuid}>
+        {getFullItem}
+      </div>
   );
 };
 
 /**
  * Render a string within a div element.
  *
  * @param {object} params - The parameters for rendering the string item.
- * @param {string} params.text - Text to be rendered
+ * @param {string} params.content - Text to be rendered
  * @param {string} params.key - The key for the rendered div element, to ensure React elements have unique keys.
  * @param {boolean} [params.isDone] - Optional. If true, the item is styled as completed.
  *
  * @returns {JSX.Element} The rendered string item.
  */
-const renderStringCell = ({text, key, isDone}: {text:string; key:string; isDone?: boolean}): JSX.Element => {
+const renderStringCell = ({content, key, isDone}: {
+
+  /**
+   * Cell's content
+   */
+  content: string;
+
+  /**
+   * Unique value for cells in table
```

Comment:
shoud it be unique for the whole table or just one row?

Reply:
Changed